src/agoradatatools/etl/transform/custom/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""Submodule for Agora Data Tools Custom Transformations"""
```

Comment:
Can you speak to why you decided to put everything in a custom folder which adds a layer of hierarchy?

Reply:
Mostly to make importing the custom transformation functions simple, and to minimize the changes needed to the codebase when a new custom transformation is contributed. without doing this, the imports on `transform/apply.py` would be either really long, or would include helper functions from the custom transformation scripts that we do not need there.

Reply:
So, I was thinking we could do something like this: transform/apply.py -> etl/apply.py

Then shifting everything from `custom` into `transform` and then the imports could look something like this:

```
from agoradatatools.etl import transform
transform.transform... 
```

Reply:
After discussion with @thomasyu888 we opted for moving the function `apply_custom_transformations` from its own script (was `apply.py`) into `process.py`, since it was not actually _doing_ any transformations itself. That freed up the `transform` module to be occupied by the custom transformation scripts, and I put the submodule there instead of in its own `transform/custom` module. We also decided to move the functions that I had originally put in `transform/utils.py` into the `etl/utils.py` file, since they were already being used in both `process.py` and the `transform` module. All of this simplifies testing, reduces a layer of hierarchy within `transform`, and simplifies imports for `process.py`.